code/modules/awaymissions/maploader/reader.dm

```diff
@@ -309,10 +321,6 @@ var/global/dmm_suite/preloader/_preloader = null
 
 	. = ..()
 
-/dmm_suite/Destroy()
-	..()
-	return QDEL_HINT_HARDDEL_NOW
-

```

Comment:
Not at all. I just copied the old PR, I didnt realize anyone had actually touched this file in the past 2 years.